azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/container/ChatCompositeViewImpl.kt

```diff
@@ -8,20 +8,26 @@ import android.widget.FrameLayout
 import androidx.compose.ui.platform.ComposeView
```

Comment:
If possible, leave ChatView the same as it is currently, but rename to something like

ChatCompositeFullscreenView()
make internal for now (since we aren't exposing the full, with navigation view).

Then we start forgetting about it in PP scope. The current one is good, doesn't need any changes to nav/etc.

We however need a new ChatCompositeView that is just message list + send bar. Can copy/paste this one and strip down all the nav stuff, extra scaffolding etc.

Should have 2 Views (1 internal/hidden/unused, current full screen, existing is fine, 1 public used, message list + send button).

Reply:
This ChatCompositeView is not exactly for Full Screen, it just holds all elements.